nixos/tests/all-tests.nix

```diff
@@ -372,6 +372,7 @@ in {
   gnome-xorg = handleTest ./gnome-xorg.nix {};
   gns3-server = handleTest ./gns3-server.nix {};
   gnupg = handleTest ./gnupg.nix {};
+  goatcounter = handleTest ./goatcounter.nix {};
```

Comment:
You could also add this test to the goatcounter package, i.e. add `nixosTests.goatcounter` to `goatcounter.passthru.tests`

Reply:
done